virttest/libvirt_xml/devices/hostdev.py

```diff
@@ -3,15 +3,16 @@
 
 http://libvirt.org/formatdomain.html#elementsHostDev
 """
-from virttest.libvirt_xml.devices import base
 from virttest.libvirt_xml import accessors
+from virttest.libvirt_xml.devices import base
+from virttest.libvirt_xml.devices import librarian
 
 
 class Hostdev(base.TypedDeviceBase):
 
     __slots__ = ('type', 'mode', 'managed', 'sgio', 'rawio',
                  'source', 'boot_order', 'readonly', 'shareable',
-                 'alias', 'model', 'teaming')
+                 'alias', 'model', 'teaming', 'rom', 'address')
```

Comment:
perhaps add simple hostdev xml under class Hostdev(base.TypedDeviceBase):
to make people understand what one typical hostdev looks like.

Reply:
Currently already there are some devices xml following up that practice. But it doesn't block approve